src/NuGetGallery/Services/ImageDomainValidator.cs

```diff
@@ -13,7 +13,7 @@ namespace NuGetGallery
     public class ImageDomainValidator: IImageDomainValidator
     {
         private static readonly TimeSpan RegexTimeout = TimeSpan.FromMinutes(1);
-        private static readonly Regex GithubBadgeUrlRegEx = new Regex("^(https|http):\\/\\/github\\.com\\/[^/]+\\/[^/]+\\/workflows\\/.*badge\\.svg", RegexOptions.IgnoreCase, RegexTimeout);
+        private static readonly Regex GithubBadgeUrlRegEx = new Regex("^(https|http):\\/\\/github\\.com\\/[^/]+\\/[^/]+\\/actions\\/workflows\\/.*badge\\.svg", RegexOptions.IgnoreCase, RegexTimeout);
```

Comment:
Can we change to actions to be optional, like (/actions/)?
```suggestion
        private static readonly Regex GithubBadgeUrlRegEx = new Regex("^(https|http):\/\/github\.com\/[^/]+\/[^/]+\/(actions\/)?workflows\/.*badge\.svg", RegexOptions.IgnoreCase, RegexTimeout);
```